sc-tools/sc-config/src/sc_config.cpp

```diff
@@ -66,10 +66,27 @@ std::string ScConfigGroup::operator[](std::string const & key) const
   if (it == m_pathKeys.cend())
     return value;
 
-  if (value[0] == '/')
-    stream << value;
-  else
-    stream << m_configPath << value;
+  std::string valueStr(value);
+  std::string::size_type start = 0;
+
+  while (true)
```

Comment:
stringstream + getline with a custom delim could be a cleaner way to implement this functionality